src/app/pages/TopicPage/HierarchicalGrid/index.tsx

```diff
@@ -58,86 +58,90 @@ const HiearchicalGrid = ({ promos, headingLevel }: Promos) => {
 
   const promoItems = promos.slice(0, 12);
   return (
-    <ul role="list" css={styles.list} data-testid="hierarchical-grid">
-      {promoItems.map((promo, i) => {
-        const duration = moment.duration(promo.duration, 'seconds');
-        const separator = ',';
-        const formattedDuration = formatDuration({ duration, separator });
-        const durationString = `${durationTranslation}, ${formattedDuration}`;
+    <div data-testid="hierarchical-grid">
+      <ul role="list" css={styles.list} data-testid="topic-promos">
```

Comment:
Note: this is to ensure consistency with the Simple Curation Grid, so that the cypress tests can use the same `data-testid` to get the number of promos in the first curation.